code401challenges/src/main/java/linked_list/LinkedList.java

```diff
@@ -0,0 +1,62 @@
+package linked_list;
+
+public class LinkedList {
+     Node head;
+
+//  Constructor creates an empty list
+    public LinkedList() {
+        this.head = null;
+    }
+
+//  Set head
+    public void setHead(int value){
+//      creates new node
+        head = new Node(value);
+    }
+
+    //Define a method called insert
+    public void insert(Node newNode){
+        if(head == null){
+            head = newNode;
+        } else {
+           newNode.next = head;
+           head = newNode;
+        }
+
+    }
+// Checks to see if a value exists in our linked list
+    public Boolean includes(Node node){
```

Comment:
Since you're just returning the primitive data type, you could have returned a `boolean` instead of a `Boolean`. While the difference doesn't matter much, `boolean`s are more space efficient